README.md

```diff
@@ -25,6 +25,10 @@ and attempts to upload the file.
 
 Install with composer by running `composer require silverstripe/mimevalidator` in the root of your SilverStripe project.
 
+Silverstripe CMS Recipe 4.6 and above include this module via `silverstripe/recipe-core`.
+Therefore, it is unnecessary to directly install this module if your project has been upgraded to,
```

Comment:
This isn't quite right - any project that directly uses a module should be declaring it in their own project's `composer.json`. If they aren't directly using it (and just relying on the core's use of it) then this applies

Reply:
Sorry, I'm not sure I'm following. That's now in core, similar to `silverstripe/assets`, for example.
Users don't have to require it explicitly on the application level anymore, if they use CMS Recipes as base for the application.